### ISSUE TITLE
Stop the forced Torch install

### ISSUE THREAD

**Issue by LSXAxeller** [josStorer/RWKV-Runner#128]
Line 130: python + " -m pip install torch==1.13.1 torchvision==0.14.1 torchaudio==0.13.1 --index-url https://download.pytorch.org/whl/cu117\n"

this line is preventing the RWKV from loading the model if I was using any different version of  Torch, which preventing me from using any version from the RWKV Runner after v1.3.0.
I am using Torch 2.0.1+cu118 which I am using with all ML projects or models on my pc since I have limited Internet Access so I can't download a different Torch version for every project I use, and the 2.0.1 was compatible with all of them, although I can't use the Custom Compiled CUDA Kernels of the Runner but not a big loss, it's still performant.

So all I asking is to make the it optional to override this installation process or skip it if another version of torch was detected, I tried to remove the line and Compile it from source but since I am familiar with GO I got a lot of errors.

And Thanks for your hard working that I can use LLMs with no need to mess with the WebUIs or Inferences.

**Comment by josStorer** [comment]
You can install these packages with any versions by yourself:
https://github.com/josStorer/RWKV-Runner/blob/master/backend-python/requirements.txt

Specify the corresponding Python interpreter path in the Settings page. Once all dependencies are met, the internal dependency installation will not be invoked

**Comment by LSXAxeller** [comment]
![image](https://github.com/josStorer/RWKV-Runner/assets/56027532/e17d2b7b-b4a7-44b4-b381-ec9d111cf5c7)

Now it's asking me to install dependencies and when I click install nothing happens

![image](https://github.com/josStorer/RWKV-Runner/assets/56027532/2bd07d79-1206-461d-82fe-acd19458aa20)

**Comment by josStorer** [comment]
```
--extra-index-url https://download.pytorch.org/whl/cu118
torch==2.0.1+cu118
torchversion...
...
...
```

**Comment by josStorer** [comment]
BTW, the Novel series models is trained on a large number of Chinese novels. For English writing, I suggest you use the World or Raven series.

**Comment by josStorer** [comment]
remove `==0.15.2`

**Comment by LSXAxeller** [comment]
Or if just torchversion without ==0.15.2

ERROR: Could not find a version that satisfies the requirement torchversion (from versions: none)
ERROR: No matching distribution found for torchversion

**Comment by QinCaii** [comment]
same here,this will only force the installation of pytorch2.0.1+cpu, it's just a piece of

**Comment by LSXAxeller** [comment]
Downloaded the Torch 1.13.1 and the Runner worked, hope no loss in performance in comparison to 2.0.1+cu118

**Comment by josStorer** [comment]
I thought your issue was already resolved. In fact, you don't have to install a specific version of PyTorch. RWKV-Runner doesn't impose any special restrictions on your Python package versions. You only need to install the packages required for your Python installation as usual.

You can start by executing the following command: `pip3 install torch torchvision torchaudio --index-url https://download.pytorch.org/whl/cu118`, just as it's mentioned on the PyTorch official website.

You can also install the following packages listed in the `requirements.txt` file:
```
--extra-index-url https://download.pytorch.org/whl/cu118
torch==2.0.1+cu118
rwkv
langchain
fastapi
uvicorn
sse-starlette
pydantic
psutil
gputil
tiktoken
ftfy
lm-dataformat
numpy
tokenizers
tqdm
cyac
```
